chirp/drivers/kenwood_d7.py

```diff
@@ -13,332 +13,3404 @@
+                        ("1380", "1380"), ("1390", "1390"), ("1400", "1400"),
+                        ("1410", "1410"), ("1420", "1420"), ("1430", "1430"),
+                        ("1440", "1440"), ("1450", "1450"), ("1460", "1460"),
+                        ("1470", "1470"), ("1480", "1480"), ("1490", "1490"),
+                        ("1500", "1500"), ("1510", "1510"), ("1520", "1520"),
+                        ("1530", "1530"), ("1540", "1540"), ("1550", "1550"),
+                        ("1560", "1560"), ("1570", "1570"), ("1580", "1580"),
+                        ("1590", "1590"), ("1600", "1600"), ("1610", "1610"),
+                        ("1620", "1620"), ("1630", "1630"), ("1640", "1640"),
+                        ("1650", "1650"), ("1660", "1660"), ("1670", "1670"),
+                        ("1680", "1680"), ("1690", "1690"), ("1700", "1700"),
+                        ("1710", "1710"), ("1720", "1720"), ("1730", "1730"),
+                        ("1740", "1740"), ("1750", "1750"), ("1760", "1760"),
+                        ("1770", "1770"), ("1780", "1780"), ("1790", "1790"),
+                        ("1800", "1800"), ("1810", "1810"), ("1820", "1820"),
+                        ("1830", "1830"), ("1840", "1840"), ("1850", "1850"),
+                        ("1860", "1860"), ("1870", "1870"), ("1880", "1880"),
+                        ("1890", "1890"), ("1900", "1900"), ("1910", "1910"),
+                        ("1920", "1920"), ("1930", "1930"), ("1940", "1940"),
+                        ("1950", "1950"), ("1960", "1960"), ("1970", "1970"),
+                        ("1980", "1980"), ("1990", "1990"), ("2000", "2000"),
+                        ("2010", "2010"), ("2020", "2020"), ("2030", "2030"),
+                        ("2040", "2040"), ("2050", "2050"), ("2060", "2060"),
+                        ("2070", "2070"), ("2080", "2080"), ("2090", "2090"),
+                        ("2100", "2100"), ("2110", "2110"), ("2120", "2120"),
+                        ("2130", "2130"), ("2140", "2140"), ("2150", "2150"),
+                        ("2160", "2160"), ("2170", "2170"), ("2180", "2180"),
+                        ("2190", "2190"), ("2200", "2200"), ("2210", "2210"),
+                        ("2220", "2220"), ("2230", "2230"), ("2240", "2240"),
+                        ("2250", "2250"), ("2260", "2260"), ("2270", "2270"),
+                        ("2280", "2280"), ("2290", "2290"), ("2300", "2300"),
+                        ("2310", "2310"), ("2320", "2320"), ("2330", "2330"),
+                        ("2340", "2340"), ("2350", "2350"), ("2360", "2360"),
+                        ("2370", "2370"), ("2380", "2380"), ("2390", "2390"),
+                        ("2400", "2400"), ("2410", "2410"), ("2420", "2420"),
+                        ("2430", "2430"), ("2440", "2440"), ("2450", "2450"),
+                        ("2460", "2460"), ("2470", "2470"), ("2480", "2480"),
+                        ("2490", "2490"), ("2500", "2500"))},
```

Comment:
We could probably write a generator to make this in a lot fewer lines of code. Looks like it's 0-2500 in steps of ten? Something like:
```
[('Off', '0000')] + [('%i' % x, '%04i' % x) for x in range(10, 2510, 10)]
```

Reply:
Perfect, I'm not a python developer, so I didn't think of that.